packages/structures/src/invites/Invite.ts

```diff
@@ -0,0 +1,221 @@
+import { type APIInvite, type APIExtendedInvite, RouteBases } from 'discord-api-types/v10';
+import { Structure } from '../Structure.js';
+import { kData, kExpiresTimestamp, kCreatedTimestamp } from '../utils/symbols.js';
+import type { Partialize } from '../utils/types.js';
+
+// TODO: use an actual dtypes type instead if it exists
```

Comment:
What type is missing?

Reply:
The one defined here. `APIInvite` but with all additional properties of `APIExtendedInvite` as optional.